packages/core/src/commands/up/index.ts

```diff
@@ -173,7 +177,7 @@ const up = async ({
     const composeArgs = calcComposeArgs({ userSpecifiedServices, debug, cwd })
     log.debug('Running compose up with args: ', composeArgs)
     await withDockerSocket(() => compose.spawnPromise(composeArgs, { stdio: 'inherit' }))
-
+    const credentials = await getUserCredentials(jwtGenerator(sshTunnelPrivateKey))
```

Comment:
I would extract this out of the `withDockerSocket` callback since the docker socket is not needed here

Reply:
It's not in withDockerSocket, it's in the scope of up(...)

Reply:
Right, sorry